examples/kustomize.jsonnet

```diff
@@ -1,5 +1,6 @@
 local kp =
-  (import 'kube-prometheus/kube-prometheus.libsonnet') + {
+  (import 'kube-prometheus/kube-prometheus.libsonnet') + { 
```

Comment:
let's revert this completely so there's no change here anymore, it looks like it's just whitespace

Reply:
79c670bcd052cee7d0dcaf786cf5ff0b5b797421 this reverts the `examples/kustomize.jsonnet` file completely.